src/app/zap-templates/zcl/data-model/chip/wifi-network-diagnostics-cluster.xml

```diff
@@ -52,11 +52,11 @@ limitations under the License.
     <code>0x0036</code>
     <define>WIFI_NETWORK_DIAGNOSTICS_CLUSTER</define>
     <description>The Wi-Fi Network Diagnostics Cluster provides a means to acquire standardized diagnostics metrics that MAY be used by a Node to assist a user or Administrative Node in diagnosing potential problems.</description>
-    <attribute side="server" code="0x00" define="BSSID" type="OCTET_STRING" length="6" writable="false" optional="false">bssid</attribute>
-    <attribute side="server" code="0x01" define="SECURITY_TYPE" type="ENUM8" writable="false" optional="false">SecurityType</attribute>
-    <attribute side="server" code="0x02" define="WIFI_VERSION" type="ENUM8" writable="false" optional="false">WiFiVersion</attribute>
-    <attribute side="server" code="0x03" define="CHANNEL_NUMBER" type="INT16U" min="0x0000" max="0xFFFF" writable="false" default="0x0000" optional="false">ChannelNumber</attribute>
-    <attribute side="server" code="0x04" define="RSSI" type="INT8S" min="-120" max="0" writable="false" default="0x00" optional="false">Rssi</attribute>
+    <attribute side="server" code="0x00" define="BSSID" type="OCTET_STRING" length="6" writable="false" isNullable="true" optional="false">bssid</attribute>
+    <attribute side="server" code="0x01" define="SECURITY_TYPE" type="ENUM8" writable="false" isNullable="true" optional="false">SecurityType</attribute>
```

Comment:
```suggestion
    <attribute side="server" code="0x01" define="SECURITY_TYPE" type="SecurityType" writable="false" isNullable="true" optional="false">SecurityType</attribute>
```

Reply:
Hit following compile error with this change

```
../../../examples/lighting-app/linux/third_party/connectedhomeip/src/lib/core/Optional.h:124:9: error: invalid conversion from ‘unsigned char’ to ‘EmberAfSecurityType’ [-fpermissive]
  124 |         new (&mValue.mData) T(std::forward<Args>(args)...);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |         |
      |         unsigned char
```

Reply:
Acked